content/protocols/index.textile

```diff
@@ -0,0 +1,57 @@
+---
+title: Protocols
+meta_description: "Clients can use the Ably network protocol adapters. This is especially useful where an Ably Client Library SDK is not available for your language of choice, or where platform resource constraints prohibit use of a Client Library SDK."
+---
+
+Ably Client Library SDKs are the recommended method for connecting to Ably because they offer support for a comprehensive set of Ably features, such as automatic "connection management":/connect, "authentication token renewal":/auth/token and "presence":/presence-occupancy.
+
+Protocol adapters offer an alternative method for connecting to Ably. The advantage to protocol adapters is that they require fewer resources in terms of memory and network overhead such as in smaller footprint devices, or on a platform where an Ably Client Library SDK isn't available such as an Arduino-based IoT wearable. The potential drawback to consider when evaluating protocol adapters is that they do not support the full set of Ably features, for example the MQTT protocol adapter does not support presence, and the SSE protocol adapter does not support automatic token renewal.
+
+A full list of Ably Client Library SDKs can be found on the "SDK page":https://ably.com/download, and features supported by each SDK can be found in the "features matrix":https://ably.com/download/sdk-feature-support-matrix.
+
+h2(#available-adapters). Available Protocol Adapters
+
+Ably has a number of fully supported adapters:
+
+* MQTT 
+* SSE 
+* AMQP
+* STOMP
+* Pusher
+* PubNub
+
+h3(#mqtt). MQTT
+
+MQTT (MQ Telemetry Transport) is a publish/subscribe, lightweight messaging protocol designed for constrained devices and low-bandwidth networks. One of the major uses of MQTT is with IoT (Internet of Things), where these principles are key to having effective communication between various devices.
+
+MQTT can also be used with Ably as a basic event broker or if we don’t have an SDK for your target platform. However, without an SDK you don’t get access to the full range of platform features and data guarantees.
+
+Read more in the "MQTT section":/protocols/mqtt
+
+h3(#sse). SSE
+
+SSE is a push technology commonly used to send unidirectional message updates or continuous data streams to a browser client. SSE aims to enhance native, cross-browser server-to-client streaming through a JavaScript API called EventSource, standardized as part of HTML5 by the World Wide Web Consortium (W3C).
+
+The Ably SSE and raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime client library, or even an MQTT library, is impractical.
+
+Read more in the "SSE section":/protocols/sse
+
+h3(#amqp). AMQP
```

Comment:
I think it might be worth linking AMQP and STOMP to the queues section so it's clear that they're used within that context. WDYT?